### PR TITLE
Major CompSci coming through...

### PR DESCRIPTION
**Pull request by Trancendence**
Starting to clean up the ugly, undocumented layout left behind by the so-called 'Software Engineers';

Server.js has been calculated to be correct by a designated Computer Science Major;

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
-	profound = require('./modules/profound.js'),
-	modules = [disable,karma,runbot,anim,magic8Ball,fawlty,profound,associate];
-	
+/** A docstring that tells us 'informative' information about server.js
+ *
+ * The Node.js serverside script to tell it what ot do to ensure we get all the
+ * magical goodness that is (Karma + Sassy) * Facebook - Hipchat = Kassy
+ *
+ * Written By:
+ * 		Matt Knox,
```
Comment (mrkno) [inline server.js]: I do prefer a full name tbh

File: server.js [hunk 2]
```diff
@@ -1,14 +1,46 @@
...
-	runbot = require('./modules/runbot.js'),
-	anim = require('./modules/anim.js'),
-	magic8Ball = require('./modules/8Ball.js'),
-	disable = require('./modules/disable.js'),
-	associate = require('./modules/associate.js'),
-	fawlty = require('./modules/fawlty.js'),
-	profound = require('./modules/profound.js'),
-	modules = [disable,karma,runbot,anim,magic8Ball,fawlty,profound,associate];
-	
+/** A docstring that tells us 'informative' information about server.js
```
Comment (mrkno) [inline server.js]: I take this wasn't intended to be left

File: server.js [hunk 3]
```diff
@@ -1,14 +1,46 @@
...
+ * magical goodness that is (Karma + Sassy) * Facebook - Hipchat = Kassy
+ *
+ * Written By:
+ * 		Matt Knox,
+ * 		Dion Woolley
+ * 		Jay Harris
+ * 		Matt Hartstonge
+ * 		(Mainly strange people)
+ *
+ * Date written: 21/07/2015
```
Comment (mrkno) [inline server.js]: Probably no point keeping a date given it's changing hourly at the moment

File: server.js [hunk 4]
```diff
@@ -1,14 +1,46 @@
...
+ *
+ * The Node.js serverside script to tell it what ot do to ensure we get all the
+ * magical goodness that is (Karma + Sassy) * Facebook - Hipchat = Kassy
+ *
+ * Written By:
+ * 		Matt Knox,
+ * 		Dion Woolley
+ * 		Jay Harris
+ * 		Matt Hartstonge
+ * 		(Mainly strange people)
```
Comment (mrkno) [inline server.js]: I like it

File: server.js [hunk 5]
```diff
@@ -1,14 +1,46 @@
...
+ *
+ * Date written: 21/07/2015
+ *
+ */
+
```
Comment (mrkno) [inline server.js]: We don't like blank lines

Comment (mrkno) [inline server.js]: Not when they are consecutive anyway

File: server.js [hunk 6]
```diff
@@ -25,12 +58,15 @@ if (typeof String.prototype.endsWith != 'function') {
 		return this.indexOf(suffix, this.length - suffix.length) !== -1;
 	};
 }
-	
-// Startup
+
+
```
Comment (mrkno) [inline server.js]: 2 consecutive blank newlines

File: server.js [hunk 7]
```diff
@@ -25,12 +58,15 @@ if (typeof String.prototype.endsWith != 'function') {
...
+// Load Optional Modules
 for (var i = 0; i < modules.length; i++) {
 	modules[i].load();
 }
 
+
```
Comment (mrkno) [inline server.js]: 2 consecutive blank newlines

**Comment by mrkno** [comment]
Fix blank newlines and merge conflict (with your own module!) and it should be ready to go

**Comment by Trancendence** [comment]
Is there a way to see the merge conflicts on Github? I can't find a diff in Github to show where it's having issues..

**Comment by Trancendence** [comment]
Well.. That's awkward. But it worked... :/

**Pull request closed without merging**